src/julia-syntax.scm

```diff
@@ -1837,13 +1837,19 @@
       `(= ,lhs ,rhs)))
 
 (define (expand-forms e)
-  (if (or (atom? e) (memq (car e) '(quote inert top core globalref outerref line module toplevel ssavalue null true false meta using import export)))
-      e
-      (let ((ex (get expand-table (car e) #f)))
-        (if ex
-            (ex e)
-            (cons (car e)
-                  (map expand-forms (cdr e)))))))
+  (cond ((or (atom? e)
+             (memq (car e) '(quote inert top core globalref outerref module toplevel ssavalue
+                             null true false meta using import export)))
+         e)
+        ((eq? (car e) 'line)
```

Comment:
This case can be added to `expand-table` below instead.

Reply:
Nice, I'll do that.